storage/internal/benchmarks/utils.go

```diff
@@ -138,10 +177,13 @@ func errorIsDeadLineExceeded(err error) bool {
 		return true
 	}
 
-	for err = errors.Unwrap(err); err != nil; {
+	err = errors.Unwrap(err)
+	for err != nil {
 		if status.Code(err) == codes.DeadlineExceeded {
 			return true
 		}
+
+		err = errors.Unwrap(err)
```

Comment:
Why is this needed?

Reply:
hm, this comment made me reconsider adding this here. Will revert this change and open a separate PR for this, as this change is due to a bug I found while writing this PR, but is not necessarily relevant here.